docs/docs/how-to/local-development/prettier.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Using Prettier With Gatsby
+---
+
+Prettier is an opinionated code formatter that auto-formats files commonly used with Gatsby. This makes your code more readable without long lists of stylistic comments polluting PR reviews, or, worse, team conflict over the right way to format code.
+
+Prettier formats JSX, CSS and variants (including Sass and styled-components), and Markdown (including MDX), all of which are commonly used on Gatsby sites.
+
+## Get started with Prettier
+
+There are two different ways to install Prettier:
+
+### Method 1: Set up with new project
+
+All official Gatsby starters come with Prettier installed. Start a new project by installing any official Gatsby starter.
+
+```sh
+gatsby new my-default-starter https://github.com/gatsbyjs/gatsby-starter-default
+```
+
+### Method 2: Add Prettier to your existing project
+
+From the root of your project, install Prettier via the command line. This will not only install Prettier on your machine, it will modify your `package.json` to add Prettier to your project's setup configuration.
+
+```
+npm install --save-dev --save-exact prettier
+```
+
+Then, create an empty config file to let editors and other tooling know you are using Prettier:
+
+```sh
+echo {}> .prettierrc.json
```

Comment:
We don't typically use shell commands for adding files, it's another layer of abstraction for new users. We should tell them to add a file to their root directory with that name.